packages/ibm-products/package.json

```diff
@@ -54,52 +54,52 @@
     "upgrade-dependencies": "npm-check-updates -u --dep dev,peer,prod --color --reject '/(carbon|^react$|^react-dom$|^chalk$|^namor)/'"
   },
   "devDependencies": {
-    "@babel/cli": "^7.23.9",
-    "@babel/core": "^7.23.9",
+    "@babel/cli": "^7.24.5",
+    "@babel/core": "^7.24.5",
     "@babel/plugin-proposal-class-properties": "^7.17.12",
-    "@babel/plugin-proposal-export-default-from": "^7.17.12",
+    "@babel/plugin-proposal-export-default-from": "^7.24.1",
     "@babel/plugin-proposal-export-namespace-from": "^7.17.12",
-    "@babel/plugin-transform-react-constant-elements": "^7.17.12",
-    "@babel/preset-env": "^7.23.9",
-    "@babel/preset-react": "^7.23.3",
-    "@babel/preset-typescript": "^7.21.5",
+    "@babel/plugin-transform-react-constant-elements": "^7.24.1",
+    "@babel/preset-env": "^7.24.5",
+    "@babel/preset-react": "^7.24.1",
+    "@babel/preset-typescript": "^7.24.1",
     "@ibm/telemetry-js-config-generator": "^1.0.3",
-    "@rollup/plugin-babel": "^6.0.0",
-    "@rollup/plugin-commonjs": "^25.0.0",
-    "@rollup/plugin-node-resolve": "^15.0.0",
-    "@rollup/plugin-typescript": "^11.0.0",
+    "@rollup/plugin-babel": "^6.0.4",
+    "@rollup/plugin-commonjs": "^25.0.7",
+    "@rollup/plugin-node-resolve": "^15.2.3",
+    "@rollup/plugin-typescript": "^11.1.6",
     "babel-plugin-dev-expression": "^0.2.3",
     "babel-preset-ibm-cloud-cognitive": "^0.14.40",
     "chalk": "^4.1.2",
-    "change-case": "4.1.2",
+    "change-case": "5.4.4",
```

Comment:
I noticed the ci-checks [complaining](https://github.com/carbon-design-system/ibm-products/actions/runs/9089835983/job/24981641534?pr=5228#step:6:87) about a missing export for `paramCase`. It seems in v5 `paramCase` has been renamed to `kebabCase` ([release notes](https://github.com/blakeembrey/change-case/releases/tag/change-case%405.0.0)).

Usage docs have changed as well, although I'm not sure if it requires a rewrite to the existing imports. According to the latest docs, it should look something like

```js
import * as changeCase from 'change-case';

const substitutions = {
  DISPLAY_NAME: changeCase.pascalCase(name),
```